src/MangaJet/app/src/main/java/com/mangajet/mangajet/data/Manga.kt

```diff
@@ -0,0 +1,6 @@
+package com.mangajet.mangajet.data
+
+//dummy class which Implementation will be done later
```

Comment:
It will be better if you write comment about what it will be, not about what it is now